SCAutolib/controller.py

```diff
@@ -171,11 +199,16 @@ def setup_local_ca(self, force: bool = False):
             raise exceptions.SCAutolibWrongConfig(msg)
 
         ca_dir: Path = self.lib_conf["ca"]["local_ca"]["dir"]
-        self.local_ca = local_ca_factory(ca_dir, force)
+        self.local_ca = local_ca_factory(path=ca_dir, force=force, create=True)
         logger.info(f"Local CA is configured in {ca_dir}")
 
         dump_to_json(self.local_ca)
 
+    def setup_custom_ca(self, card_data: dict):
+        if card_data["card_type"] == "physical":
```

Comment:
not sure what the flow for this method, but seems like card_data should be some object with `card_type` attribute

Reply:
Earlier in controller a .json file is deserialized  and provided as dictionary to _validate_configuraton that returns another dictionary  called 'self.lib_conf'. Part of this dictionary is is provided as a parameter here and I think approaching it as dict is correct here.